CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - Evaluate interpolated strings during constant propagation (#3127)
 - Fixed #3084 - Semgrep will report an InvalidRuleSchemaError for dictionaries with duplicate key names.
 - Basic type inference also for implicit variable declarations (Python, Ruby, PHP, and JS)
+- JavaScript parsing: [Support decorators on
```

Comment:
Since we cut a release will need to move this changelog entry

Reply:
yes, sure. I was hoping to finish last night but didn't make it. The bigger priority for semgrep is the typescript upgrade anyway, which will use some of this work. It should be in the next release unless some complications arise.